src/sentry/utils/email/message_builder.py

```diff
@@ -75,6 +75,25 @@ def inline_css(value: str) -> str:
     return html
 
 
+def message_to_dict(message: EmailMultiAlternatives) -> dict[str, Any]:
+    return {
+        "subject": message.subject,
+        "body": message.body,
+        "from_email": message.from_email,
+        "to": message.to,
+        "cc": message.cc,
+        "bcc": message.bcc,
+        "attachments": message.attachments,
+        "headers": message.extra_headers,
+        "reply_to": message.reply_to,
+        "alternatives": message.alternatives,
```

Comment:
Talked with Evan and `attachments` and `alternatives` are named tuples that will json encode safely.